lib/Data.js

```diff
@@ -34,21 +36,17 @@ export class Suite {
     return this.tests.concat(childSuiteTests)
   }
 
-  get runtime () {
-    var status = this.status
-
-    if (status === 'skipped' || status === undefined) {
+  getRuntime () {
```

Comment:
Ok, normally I would rename it to `_getStatus()`so that I will notify it is something internal.